src/v2/controllers/FormController.js

```diff
@@ -165,6 +167,22 @@ export default Controller.extend({
       return;
     }
 
+    const identifier = model.get('identifier');
```

Comment:
I'd rather to change json result instead of model itself.
```
const requestValue = model.toJSON();
if requestValue.identify then
  ...
idx.proceed(formName, requestValue);
```